authors/apps/articles/tests/test_comments.py

```diff
@@ -0,0 +1,199 @@
+import json
+from django.urls import reverse
+from rest_framework.views import status
+from rest_framework.test import APITestCase, APIClient
+
+
+class CommentsTestCase(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.signup_url = reverse('authentication:auth-register')
+        self.article_listcreate = reverse('articles:articles-listcreate')
+        self.signup_data = {
+                        "user": {
+                            "username": "maggie123",
+                            "email": "margaret.chege@andela.com",
+                            "password": "Pass@123"
+                        }
+                        }
+        self.reply_to_comment = {"comment": {
```

Comment:
Why isn't the request to the thread endpoint as simple as the others? It shouldn't need to have the `comment:`